pkg/internal/diskutil_test.go

```diff
@@ -113,16 +138,17 @@ func TestListBlockDevices(t *testing.T) {
 		},
 		{
 			label:             "Case 3",
```

Comment:
please change labels to be descriptive everywhere. "Case 1".. etc are not good enough labels.

Reply:
added descriptive labels.